alws/utils/copr.py

```diff
@@ -22,10 +22,14 @@ def generate_repo_config(
 ) -> str:
     # we should clean "http" protocol from host url
     clean_host_name = re.sub(r'^(http|https)://', '', settings.pulp_host)
+    repo_url = re.sub(
+        '(x86_64|aarch64|ppc64le|i386|i686|s390x)', '$basearch', repo.url
```

Comment:
that won't work for products that contain arches in their names
```
In [5]: url = 'https://build.almalinux.org/pulp/content/builds/almalinux-8-beta-s390x-AlmaLinux-8-s390x-dr/'

In [6]: re.sub('(x86_64|aarch64|ppc64le|i386|i686|s390x)', '$basearch', url)
Out[6]: 'https://build.almalinux.org/pulp/content/builds/almalinux-8-beta-$basearch-AlmaLinux-8-$basearch-dr/'
```

Reply:
Valid point, will fix it

Reply:
Fixed